conf/test_full.config

```diff
@@ -14,14 +14,14 @@ params {
     config_profile_name        = 'Full test profile'
     config_profile_description = 'Full test dataset to check pipeline function'
 
-    input                    = 'assets/samplesheet.csv'
-    genome_sheet             = "assets/genomesheet.csv"
-        file_prefix              = 'coproid'
+    input                    = "assets/samplesheet_full.csv"
+    genome_sheet             = "assets/genomesheet_full.csv"
+    file_prefix              = 'coproid'
     sam2lca_identity         = 0.8
     sam2lca_acc2tax          = 'adnamap' //'adnamap' for db build, 'nucl' for default
     kraken2_db               = 'https://github.com/nf-core/test-datasets/raw/coproid/kraken.tar.gz'
-    sp_sources               = 'https://github.com/nf-core/test-datasets/raw/coproid/sourcepredict/test_sources.csv'
-    sp_labels                = 'https://github.com/nf-core/test-datasets/raw/coproid/sourcepredict/test_labels.csv'
+    sp_sources               = 'https://raw.githubusercontent.com/maxibor/sourcepredict/master/data/modern_gut_microbiomes_sources.csv'
```

Comment:
These will need to be hosted on the coproID repository itself. Or in the nf-core test datasets repo.